packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -125,6 +129,9 @@ class WebView extends JavaObject {
   /// The [WebSettings] object used to control the settings for this WebView.
   late final WebSettings settings = WebSettings(this);
 
+  /// The [ScrollChangedCallback] object used to listen for scroll changed events.
+  late ScrollChangedCallback? onScrollChanged;
```

Comment:
To follow https://dart.dev/guides/language/effective-dart/design#prefer-inline-function-types-over-typedefs, you can remove the `ScrollChangedCallback` typedef and just use `void Function(
    int left, int top, int oldLeft, int oldTop)`.